cmd/stepsArgo.go

```diff
@@ -89,18 +94,19 @@ func awaitGitlab() {
 	}
 }
 
-func produceGitlabTokens(){
+func produceGitlabTokens() {
 	//TODO: Should this step be skipped if already executed?
-	config, err := clientcmd.BuildConfigFromFlags("", kubeconfigPath)
+	config := configs.ReadConfig()
```

Comment:
As mentioned before, maybe move up to the root command or main to avoid using the same logic all over the place.